tests/basic_env.py

```diff
@@ -146,6 +147,7 @@ def setUp(self):
         logger.setLevel("DEBUG")
 
 
+# TODO: refactor? Looks like this one is only used once
```

Comment:
code style: we don't keep XXX and TODOs in code. It's better to create an issue and prioritize it. if are not going to resolve this in this PR at least.